### PR TITLE
360 add table name validation

### PR DESCRIPTION
**Pull request by andrey-katz-systemorph**
Add table validation to the importers

### CODE REVIEW

**Comment by dcolleoni** [comment]
Complete the PR on validation refactoring and merge develop in this one. Otherwise, all good.

File: ifrs17/Constants/Validations.ipynb [hunk 1]
```diff
@@ -45,7 +45,7 @@
             "outputs": []
         },
         {
-            "cell_type": "code",
+            "cell_type": "markdown",
```
Comment (dcolleoni) [inline ifrs17/Constants/Validations.ipynb:48]: this is wrong. It looks like a merge conflict.

File: ifrs17/Constants/Validations.ipynb [hunk 2]
```diff
@@ -88,11 +88,7 @@
                 "\n    };",
                 "\n    ",
                 "\n}"
-            ],
-            "metadata": {},
-            "execution_count": 0,
-            "outputs": []
-        },
+
```
Comment (dcolleoni) [inline ifrs17/Constants/Validations.ipynb:91]: also this extra line. It's better to remove it...

**Review by dcolleoni — APPROVED** [review]

File: ifrs17/Constants/Validations.ipynb (hunk 1 above)
Comment (andrey-katz-systemorph) [reply ifrs17/Constants/Validations.ipynb]: Yes, absolutely right. To be on the safe side, I redid now the merger from the scratch. I think it is correct now

File: ifrs17/Constants/Validations.ipynb (hunk 2 above)
Comment (andrey-katz-systemorph) [reply ifrs17/Constants/Validations.ipynb]: gone now after the correct merger has been properly redone.

**Pull request merged**